pkg/kn/commands/service/service_create_test.go

```diff
@@ -118,7 +134,7 @@ func TestServiceCreateImage(t *testing.T) {
 
 func TestServiceCreateImageSync(t *testing.T) {
 	action, created, output, err := fakeServiceCreate([]string{
-		"service", "create", "foo", "--image", "gcr.io/foo/bar:baz"}, true, true)
+		"service", "create", "foo", "--image", "gcr.io/foo/bar:baz"}, false, true)
```

Comment:
@rhuss : can you please help me explain why `nrGetCalled` is used ?

For test case `	[]string{"service", "create", "foo", "--image", "gcr.io/foo/bar:baz"}, false, true)`, shouldn't following conditional fail? (as nrGetCalled should be 1)
```
			if withExistingService || (sync && nrGetCalled > 1) {
```